crates/lib3h/tests/test_suites/two_basic.rs

```diff
@@ -100,7 +101,11 @@ fn test_send_message(alex: &mut NodeMock, billy: &mut NodeMock) {
     let req_id = alex.send_direct_message(&BILLY_AGENT_ID, "wah".as_bytes().to_vec());
     let (did_work, srv_msg_list) = alex.process().unwrap();
     assert!(did_work);
-    assert_eq!(srv_msg_list.len(), 0);
+    assert_eq!(srv_msg_list.len(), 1);
```

Comment:
Why did the code previously return no messages? Was the send direct message code only half written?

Reply:
That part of the spec is unspecified. I'm pushing for every request from Core should get a response from Lib3h.